### DISCUSSION TITLE
Version 1.9.1

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#1531] [Changelog]
Version 1.9.1 fixes one bug.

- #1529 Scratch Addons cannot be launched on non-English configuration. (prio-1)

**Comment by apple502j** [comment]
Released on February 9th, 2021

**Comment by cobaltt7** [comment]
The version I got from the store is still 1.8.1...
________________________________

**Reply by TheColaber** [reply]
Pending on Chrome, Released on FF.
